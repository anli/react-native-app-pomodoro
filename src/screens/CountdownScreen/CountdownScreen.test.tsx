import React from 'react';
import renderer from 'react-test-renderer';
import { CountdownScreen } from '.';

describe('GIVEN I am "user type" "any"', () => {
  describe('WHEN I am at "Countdown Screen"', () => {
    const tree = renderer.create(<CountdownScreen />).toJSON();

    it('THEN I Should see "default UI"', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
