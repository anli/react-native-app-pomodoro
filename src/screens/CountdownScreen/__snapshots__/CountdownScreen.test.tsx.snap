// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN I am "user type" "any" WHEN I am at "Countdown Screen" THEN I Should see "default UI" 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 64,
        },
      ]
    }
  >
    25:00
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#6200ee",
        "borderColor": "transparent",
        "borderRadius": 4,
        "borderStyle": "solid",
        "borderWidth": 0,
        "elevation": 2,
        "marginTop": 20,
        "minWidth": 64,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 0.75,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 1.5,
      }
    }
  >
    <View
      accessibilityRole="button"
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "borderRadius": 4,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
      >
        <Text
          numberOfLines={1}
          style={
            Array [
              Object {
                "color": "#000000",
                "fontFamily": "Helvetica Neue",
                "textAlign": "left",
                "writingDirection": "ltr",
              },
              Array [
                Object {
                  "letterSpacing": 1,
                  "marginHorizontal": 16,
                  "marginVertical": 9,
                  "textAlign": "center",
                },
                undefined,
                Object {
                  "color": "#ffffff",
                  "fontFamily": "HelveticaNeue-Medium",
                },
                Object {
                  "fontFamily": "HelveticaNeue-Medium",
                },
              ],
            ]
          }
        >
          Start
        </Text>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
